.github/workflows/validate_company_industry.yml

```diff
@@ -2,6 +2,10 @@ name: Validate Company Industry
 
 on:
   pull_request:
+  push:
+    branches:
+      - main
```

Comment:
I'm not sure if this is FULLY needed but I added it for good measure just in case.